drivers/auxdisplay/auxdisplay_serlcd.c

```diff
@@ -413,19 +407,19 @@ static const struct auxdisplay_driver_api auxdisplay_serlcd_auxdisplay_api = {
 
 #define AUXDISPLAY_SERLCD_INST(inst)                                                               \
 	static const struct auxdisplay_serlcd_config auxdisplay_serlcd_config_##inst = {           \
-		.capabilities = {                                                                  \
-				.columns = DT_INST_PROP(inst, columns),                            \
-				.rows = DT_INST_PROP(inst, rows),                                  \
-				.mode = 0,                                                         \
-				.brightness.minimum = AUXDISPLAY_LIGHT_NOT_SUPPORTED,              \
-				.brightness.maximum = AUXDISPLAY_LIGHT_NOT_SUPPORTED,              \
-				.backlight.minimum = AUXDISPLAY_LIGHT_NOT_SUPPORTED,               \
-				.backlight.maximum = AUXDISPLAY_LIGHT_NOT_SUPPORTED,               \
-				.custom_characters = SERLCD_CUSTOM_CHAR_MAX_COUNT,                 \
-				.custom_character_width = SERLCD_CUSTOM_CHAR_WIDTH,                \
-				.custom_character_height = SERLCD_CUSTOM_CHAR_HEIGHT,              \
-			},                                                                         \
-		.bus = I2C_DT_SPEC_INST_GET(inst)};                                                \
+		.capabilities = {.columns = DT_INST_PROP(inst, columns),                           \
+				 .rows = DT_INST_PROP(inst, rows),                                 \
+				 .mode = 0,                                                        \
+				 .brightness.minimum = AUXDISPLAY_LIGHT_NOT_SUPPORTED,             \
+				 .brightness.maximum = AUXDISPLAY_LIGHT_NOT_SUPPORTED,             \
+				 .backlight.minimum = AUXDISPLAY_LIGHT_NOT_SUPPORTED,              \
+				 .backlight.maximum = AUXDISPLAY_LIGHT_NOT_SUPPORTED,              \
+				 .custom_characters = SERLCD_CUSTOM_CHAR_MAX_COUNT,                \
+				 .custom_character_width = SERLCD_CUSTOM_CHAR_WIDTH,               \
+				 .custom_character_height = SERLCD_CUSTOM_CHAR_HEIGHT},            \
+		.bus = I2C_DT_SPEC_INST_GET(inst),                                                 \
+		.command_delay_ms = DT_INST_PROP(inst, command_delay_ms),                          \
+		.special_command_delay_ms = DT_INST_PROP(inst, special_command_delay_ms)};         \
```

Comment:
almost :-)

```c
        static const struct auxdisplay_serlcd_config auxdisplay_serlcd_config_##inst = {           \
                .capabilities = {                                                                  \
                        .columns = DT_INST_PROP(inst, columns),                                    \
                        .rows = DT_INST_PROP(inst, rows),                                          \
                        .mode = 0,                                                                 \
                        .brightness.minimum = AUXDISPLAY_LIGHT_NOT_SUPPORTED,                      \
                        .brightness.maximum = AUXDISPLAY_LIGHT_NOT_SUPPORTED,                      \
                        .backlight.minimum = AUXDISPLAY_LIGHT_NOT_SUPPORTED,                       \
                        .backlight.maximum = AUXDISPLAY_LIGHT_NOT_SUPPORTED,                       \
                        .custom_characters = SERLCD_CUSTOM_CHAR_MAX_COUNT,                         \
                        .custom_character_width = SERLCD_CUSTOM_CHAR_WIDTH,                        \
                        .custom_character_height = SERLCD_CUSTOM_CHAR_HEIGHT,                      \
                },                                                                                 \
                .bus = I2C_DT_SPEC_INST_GET(inst),                                                 \
                .command_delay_ms = DT_INST_PROP(inst, command_delay_ms),                          \
                .special_command_delay_ms = DT_INST_PROP(inst, special_command_delay_ms),          \
        };                                                                                         \
```

Reply:
I'm really not seeing what you're seeing.
I reverted my changes, only changed the following:

```c
.bus = I2C_DT_SPEC_INST_GET(inst)};                                                \
```

to

```c
.bus = I2C_DT_SPEC_INST_GET(inst),                                                \
.command_delay_ms = DT_INST_PROP(inst, command_delay_ms), \
.special_command_delay_ms = DT_INST_PROP(inst, special_command_delay_ms)); \
```

And then using [clang-format](https://docs.zephyrproject.org/latest/contribute/guidelines.html#clang-format) I'm fortmatting before commiting.
I'm not editing manually.

If there's a change to the formatting - this needs to be updated in the official documention.

I'm not manually formatting this.

Reply:
But this is exacly what I had orginally!

Reply:
uh, if it is then I messed up, anyway, looks good now, thanks

Reply:
wait --- it does, the original bracket, hangon

Reply:
so many commits and discussions on `{`  ... and I didn't even do it manually but with clang-format ...

Reply:
Yes, I'm asking you to NOT clang-format that block.